atomic_reactor/utils/cachito.py

```diff
@@ -147,14 +148,22 @@ def wait_for_request(
 
             # All other states are expected to be transient and are not checked.
 
-            elapsed = time.time() - start_time
+            # If "last_updated_value" does not match the "updated" value of the
+            # request from Cachito, then we know Cachito has performed some work
+            # since the last check, so the timer resets.
+            if last_updated_value is None or last_updated_value != response_json['updated']:
+                last_updated_value = response_json['updated']
+                last_update_time = time.time()
+
+            elapsed = time.time() - last_update_time
```

Comment:
This will cause a failure if Cachito doesn't return an `updated` value initially.

Reply:
It won't because `last_update_time` always gets set to `time.time()` on the first run because of `if last_updated_value`.